job-server/test/spark.jobserver/WebApiDataRoutesSpec.scala

```diff
@@ -0,0 +1,59 @@
+package spark.jobserver

```

Comment:
@noorul The class under test is called "DataRoutes.scala", so "DataRoutesSpec.scala" fits, doesn't it? Do you want me to rename both classes?